tests/integration/IntegrationTests.cmake

```
@@ -27,6 +27,7 @@ foreach(test ${INTEGRATION_TESTS_LIST})
 		set(SET_TEST_PROPS
 	"set_tests_properties([==[${test}]==] PROPERTIES
 		WORKING_DIRECTORY \"${CMAKE_CURRENT_SOURCE_DIR}\"
+		TIMEOUT 15
```

```suggestion
		TIMEOUT 60
```

If tests finish in 10 seconds on a typical day, then I like to have a bit of safety margin for a bad day.

---

I do agree that 1500 seconds is too long.

---

30 seconds might also be okay.

---

```suggestion
		TIMEOUT 30
```